en/flight_controller/pixhawk4_mini.md

```diff
@@ -33,6 +33,7 @@ The *Pixhawk<sup>&reg;</sup> 4 Mini* autopilot is designed for engineers and hob
   * Servo Rail Input: 0~24V
   * Max current sensing: 120A
 * Weight and Dimensions:
+  * Weight: 37.2g
```

Comment:
Thanks! How sure are you of this value?
I can't find anything official, but there are other values quoted publically https://www.getfpv.com/holybro-pixhawk-4-mini-w-pm06-v2.html

Reply:
Holybro updated their pdf datasheet.